mlflow/tracking/fluent.py

```diff
@@ -1340,6 +1344,7 @@ def setup_autologging(module):
         try:
             autolog_fn = LIBRARY_TO_AUTOLOG_FN[module.__name__]
             autologging_params = get_autologging_params(autolog_fn)
+            autologging_params.update(AUTOLOGGING_INTEGRATIONS[module.__name__])
```

Comment:
@apurva-koti what happens if I do something like:

```
mlflow.autolog()
import sklearn
mlflow.autolog(disable=True)
```

In this case, will the disable= True property be applied for sklearn autologging? It seems like the second configuration of mlflow.autolog will be ignored. This is important for the databricks autologging case.